### PR TITLE
Trials project review

### PR DESCRIPTION
**Pull request by jash-debug**
copy code from trials
Set up linters
Add descriptive readme
Review of trials project

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#header{
+    display: flex;
+    justify-content: space-between;
+
+
```
Comment (jash-debug) [inline css/style.css:5]: Please do not add unnecessary space

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    margin-right: 10px;
+    text-decoration: none;
+}
+
+.nav-link:hover{
+    background-color: violet;
+    border-radius: 15%;
+    padding-left: 15px;
+    padding-right: 15px;
+
```
Comment (jash-debug) [inline css/style.css:19]: Please, as recommended among the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), kindly avoid the unnecessary use of blank lines in your code. This offers better readability to your code. This has many occurrences in your code.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <header id="header">
+      <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" style="width: 20em;" alt="">
```
Comment (jash-debug) [inline index.html:11]: Please CSS inline styles should not be used, move styles to an external CSS file

File: index.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    </form>
+
+   <div class="video">
+    <iframe id="video" width="560" height="315" src="https://www.youtube.com/embed/y8Yv4pnO7qc" 
+    title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+   </div>
+   
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+
```
Comment (jash-debug) [inline index.html:30]: Please, as recommended among the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), kindly avoid the unnecessary use of blank lines in your code. This offers better readability to your code. This has many occurrences in your code.

**Review by jash-debug — COMMENTED** [review]
Hi @jash-debug,
Good job so far 👍🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

❌   linter errors
✔️  reviewed project run on your machine
Used best practices
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Kossi-stack — COMMENTED** [review]
## Status: Approved ✔️

**Hi @jash-debug,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**